src/Core/Content/Property/Aggregate/PropertyGroupTranslation/PropertyGroupTranslationDefinition.php

```diff
@@ -39,6 +40,7 @@ protected function defineFields(): FieldCollection
         return new FieldCollection([
             (new StringField('name', 'name'))->addFlags(new Required()),
             new LongTextField('description', 'description'),
+            new IntField('position', 'position'),
```

Comment:
please define a default in the getDefaults function.

Reply:
Implemented getDefaults.